### PR TITLE
Added hex encoding and specs to test against the PHP reference implementation

### PR DESCRIPTION
**Pull request by shaicoleman**

### CODE REVIEW

File: lib/paseto/local.rb [hunk 1]
```diff
@@ -28,6 +32,10 @@ def encode64
           Paseto.encode64(@key)
         end
 
+        def encode_hex
+          Paseto.encode_hex(@key)
```
Comment (mguymon) [inline lib/paseto/local.rb]: Is it possible to add a simple test for the encode_hex?

**Review by mguymon — CHANGES_REQUESTED** [review]

**Comment by mguymon** [comment]
I have a love / hate relationship with rubocop

**Review by mguymon — APPROVED** [review]
Looks good, ill push it out as 0.4.1

**Comment by shaicoleman** [comment]
I've added a fix, so `Paseto::V2::Public::SecretKey.generate.encode_hex` works

**Comment by mguymon** [comment]
released in https://rubygems.org/gems/paseto/versions/0.4.1

**Pull request merged**